### PR TITLE
handle duplicate rows in RADIUS accounting

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Makes sure only one row will be return in the selects of the accounting procedures.
Adds acctuniqueid unicity enforcing in acct_update
# Impacts

RADIUS accounting
# Delete branch after merge

YES
# NEWS file entries
## Bug Fixes
- Fix issues where the accounting procedure would fail due to multiple rows being returned inside it.

### CODE REVIEW

File: db/pf-schema-X.Y.Z.sql [hunk 1]
```diff
@@ -850,6 +747,126 @@ BEGIN
...
+        acctinputoctets = p_acctinputoctets,
+        acctoutputoctets = p_acctoutputoctets,
+        acctupdatetime = p_timestamp,
+        acctinterval = timestampdiff( second, Previous_AcctUpdate_Time,  p_timestamp  )
+    WHERE acctuniqueid = p_acctuniqueid 
+    AND (acctstoptime IS NULL OR acctstoptime = 0);
+
+  END IF;
+
+  # Update record with new traffic
```
Comment (louismunro) [inline db/pf-schema-X.Y.Z.sql]: Like repeating yourself much?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 2]
```diff
@@ -1,3 +1,292 @@
...
+        acctinputoctets = p_acctinputoctets,
+        acctoutputoctets = p_acctoutputoctets,
+        acctupdatetime = p_timestamp,
+        acctinterval = timestampdiff( second, Previous_AcctUpdate_Time,  p_timestamp  )
+    WHERE acctuniqueid = p_acctuniqueid 
+    AND (acctstoptime IS NULL OR acctstoptime = 0);
+
+  END IF;
+
+  # Update record with new traffic
```
Comment (louismunro) [inline db/upgrade-X.X.X-X.Y.Z.sql]: here too?

**Comment by julsemaan** [comment]
re-review

**Comment by louismunro** [comment]
LGTM. merging.

**Pull request merged**